gptel.el

```diff
@@ -902,12 +903,16 @@ Note: This will move the cursor."
   (or (alist-get major-mode gptel-response-prefix-alist) ""))
 
 (defsubst gptel--trim-prefixes (s)
-  "Remove prompt/response prefixes from string S."
-  (string-trim s
-   (format "[\t\r\n ]*\\(?:%s\\)?[\t\r\n ]*"
-             (regexp-quote (gptel-prompt-prefix-string)))
-   (format "[\t\r\n ]*\\(?:%s\\)?[\t\r\n ]*"
-           (regexp-quote (gptel-response-prefix-string)))))
+  "Remove prompt/response prefixes from string S.
+Return nil if string collapses to empty string."
+  (let ((trimmed  (string-trim
+                   (string-trim s)
```

Comment:
Why the double `string-trim`?  The inner `string-trim` can cause this operation to fail when the prefix-strings have spaces, such as when the response string is set to `Response: ` (with the trailing space).